keep-ui/utils/hooks/usePresets.ts

```diff
@@ -1,16 +1,27 @@
 import { Preset } from "app/alerts/models";
 import { useSession } from "next-auth/react";
+import { useSearchParams } from "next/navigation";
+import { useState } from "react";
 import useSWR, { SWRConfiguration } from "swr";
 import { getApiURL } from "utils/apiUrl";
 import { fetcher } from "utils/fetcher";
 
-export const usePresets = (options?: SWRConfiguration) => {
+export const usePresets = () => {
   const apiUrl = getApiURL();
   const { data: session } = useSession();
+  const searchParams = useSearchParams()!;
 
-  return useSWR<Preset[]>(
-    () => (session ? `${apiUrl}/preset` : null),
-    async (url) => fetcher(url, session?.accessToken),
-    options
-  );
+  const getCurrentPreset = () => {
+    return searchParams?.get("selectedPreset") || "Feed";
+  };
```

Comment:
`getCurrentPreset` isn't using a `useSWR` hook. This can easily live in `alerts.tsx`